### ISSUE TITLE
The injection process seems to be happening (for new game processes) even when deactivated, and when GeDoSaToTool.exe isn't running

### ISSUE THREAD

**Issue by amasover** [PeterTh/gedosato#385]
I spent a few hours tonight troubleshooting a conflict between GeDoSaTo (v0.21.2136) and FFXIV dx11. I installed GeDoSaTo in order to fix some things with FFXIII, and I had been running GeDoSaTo only when I ran FFXIII. Tonight I tried running FFXIV for the first time in awhile and the game crashed (only when set to dx11 mode, but not in dx9 mode) on startup almost every time (access violation in dxgi.dll). 

Initially I assumed that it was a driver issue because:
- GeDoSaTo was set to deactivated
- "start with windows" was unchecked (I even rebooted after unchecking that)
- I had exited out of GeDoSaToTool.exe, and confirmed that the process was no longer running.

However, I found that I could only reliably make the FFXIV crash go away by either:
- Renaming the GeDoSaTo folder (I have it in C:/GeDoSaTo if that matters)
- Removing FFXIV from the whitelist in GeDoSaTo.

This is on Windows 10, btw.

Is this a bug, or intended behavior? And could someone explain how GeDoSaTo is being injected when there is no running process to inject it? Thanks!

**Comment by XenHat** [comment]
Wow, this could be a problem if people use GeDoSato on GTA V and suddenly goes online. Rockstar is very ban-happy with dll hooks (I got banned for using an updated Direct X dll from the Windows SDK once)

**Comment by Ofunniku** [comment]
I can confirm that this is indeed happening, shim64.dll is being hooked EVERYWHERE, even on things not on the whitelist.

unlocking the dll and deleting it seems to fix the issue, but I need to know how to keep this from happening altogether.

**Comment by vertexgamer** [comment]
Confirmed, shin64.dll just hooks to everything causing crashes on some 3d accellerated softwares like trackmania 2020, rage 2, unigine superposition.
@Ofunniku btw have you found a solution? Apart from deleting it every time

**Comment by Proutbedaine** [comment]
Yeah can confirm, it's insane, it does hook to *everything* and I only noticed because I tried to move the installation folder and it gave me an error saying it was in use. 
And like the others, even though nothing other than the defaults are on the whitelist, and that "startup with windows" is unchecked, and that I open it > deactivate it > close it. 
(I haven't used it in about 2 years.)

Here's screenshots of what it's seemingly hooked to.

![Image](https://github.com/user-attachments/assets/555c1cdf-42bb-4570-b783-b5f78d7a7f7f)

![Image](https://github.com/user-attachments/assets/9285be90-4e07-44ec-b027-3db47c9e91f3)

![Image](https://github.com/user-attachments/assets/607c09aa-72a3-494a-82b9-b9171e6aaf1c)

![Image](https://github.com/user-attachments/assets/829b4875-08e8-4ca4-bda3-90d229056228)

![Image](https://github.com/user-attachments/assets/41d85e0b-23ed-419d-a802-f7a3e49d2161)

![Image](https://github.com/user-attachments/assets/74e548b6-d271-4f5d-a534-4311afa54387)

It's crazy. 
Even the File Locksmith itself lmao.
